repository-data/webfiles/src/main/resources/site/freemarker/publicationsystem/macro/resource-image.ftl

```diff
@@ -0,0 +1,36 @@
+<#ftl output_format="HTML">
+
+<#macro resourceImage file>
+    <#if file?ends_with(".xlsx") || file?ends_with(".xls")>
+        <svg width="24" xmlns="http://www.w3.org/2000/svg" class="media__img" viewBox="0 0 100 100">
```

Comment:
please, use the same approach as in styleguide: `<img src=" .../icon-xls.svg" width="24" alt="XLS">`